core/services/fluxmonitor/flux_monitor.go

```diff
@@ -576,6 +576,12 @@ func (p *PollingDeviationChecker) respondToNewRoundLog(log *contracts.LogNewRoun
 	if !roundState.EligibleToSubmit {
 		logger.Infow("Ignoring new round request: not eligible to submit", p.loggerFieldsForNewRound(log)...)
 		return
+	} else if roundState.AvailableFunds.Cmp(roundState.PaymentAmount) < 0 {
+		logger.Infow("Ignoring new round request: aggregator is underfunded", p.loggerFieldsForNewRound(log)...)
+		return
+	} else if roundState.PaymentAmount.Cmp(p.store.Config.MinimumContractPayment()) < 0 {
+		logger.Infow("Ignoring new round request: round payment amount < minimum contract payment", p.loggerFieldsForNewRound(log)...)
+		return
 	}
```

Comment:
would it be possible to remove the duplication of these checks?  Its pretty error prone

Reply:
I don't think it's the best idea -- the log messages are different, and this logging is currently very important as we observe behavior in live environments.